app/views/govuk_component/document_footer.raw.html.erb

```diff
@@ -40,7 +40,11 @@
     <% if history.any? %>
       <div class="change-notes" data-module="toggle">
         <p>
-          <a href="#full-history" data-controls="full-history" data-expanded="false">
+          <a href="#full-history" data-controls="full-history"
+                                  data-expanded="false"
+                                  data-track-category="Content History"
+                                  data-track-action="fullPageHistoryOpened"
+                                  data-track-label="full-history">
```

Comment:
Three different styles here:
* Words with spaces for category
* CamelCase for action
* Hyphenated for label

Could they all be hyphenated?